JetAnalyzers/bin/jet_response_fitter_x.cc

```diff
@@ -193,46 +198,56 @@ int main(int argc,char**argv)
         continue;
       }
 
-      //if(verbose>0) cout << "Attempting to fit " << histname << " ... " << endl;
+      //
+      // write all the histogram used later for tau calibration study
+      //
+      if( fittype==2 && (histname.find("RelRsp_JetEta")==string::npos || histname.find("_RefPt")==string::npos) ) {
+	hrsp->Write(); 
+	continue; 
+      }
 
+      //if(verbose>0) cout << "Attempting to fit " << histname << " ... " << endl;
```

Comment:
Can be removed.

Le 27/04/2016 02:18, Alexx Perloff a écrit :
>
> In JetAnalyzers/bin/jet_response_fitter_x.cc 
> <https://github.com/cms-jet/JetMETAnalysis/pull/10#discussion_r61180964>:
>
> >
> > +      //if(verbose>0) cout << "Attempting to fit " << histname << " ... " << endl;
>
> Is this comment necessary or can it be removed?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-jet/JetMETAnalysis/pull/10/files/6c09ed51401443d614ab03918245241673c2ddb0#r61180964>
>